### DISCUSSION TITLE
Change division symbol

### DISCUSSION THREAD

**Discussion by aijony** [coalton-lang/coalton#458] [Ideas and Proposals]
I find that a majority of the time when I use `/` I don't need a polymorphic output. For example, this code is needs an extra type deceleration:

`(coalton (the Fraction (/ (/ 1 2) (the Fraction 3))))`

One can imagine how nested division quickly becomes cumbersome. I propose we change the function for `Dividable` to `divide` and make a `(define / divide)` of type `(Dividable :a :a) => :a -> :a -> :a`. Now the same code would be:

`(coalton (the Fraction (/ (divide 1 2) 3))))` or while casting 1 & 2 as fractions,  `(coalton (the Fraction (/ (/ 1 2) 3)))`

Unless anyone has objections, I can make a pull request.

**Comment by stylewarning** [comment]
I'm on board with this idea. It'll be a bit confusing but we can try it. Maybe `general/` instead of `divide`, just to keep with the pattern?

**Comment by stylewarning** [comment]
FWIW to those reading along, this was ultimately implemented in the standard library!
